src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -199,17 +205,19 @@ We support
 
         private IEnumerable<IConfigurationSection> GetFeatureDefinitionSections()
         {
-            const string FeatureManagementSectionName = "FeatureManagement";
+            //
+            // Look for feature definitions under the "FeatureManagement" section
+            IConfigurationSection featureManagementConfigurationSection = _configuration.GetSection(FeatureManagementSectionName);
 
-            if (_configuration.GetChildren().Any(s => s.Key.Equals(FeatureManagementSectionName, StringComparison.OrdinalIgnoreCase)))
+            if (featureManagementConfigurationSection.Exists())
             {
-                //
-                // Look for feature definitions under the "FeatureManagement" section
-                return _configuration.GetSection(FeatureManagementSectionName).GetChildren();
+                return featureManagementConfigurationSection.GetChildren();
```

Comment:
Can we call GetChildren even if the section doesn't exist? Just thinking that if so, we can shorten this whole thing to

```
return _configuration
    .GetSection(FeatureManagementSectionName)
    .GetChildren();
```

Reply:
I just checked, yes that works. However we won't use it here if we keep the log.

Reply:
Okay, resolving.